### PR TITLE
Fix background color of testimonial in dark mode

### PR DESCRIPTION
**Pull request by sdepold**
image

### CODE REVIEW

File: src/components/homepage-users.module.scss [hunk 1]
```diff
@@ -46,7 +46,7 @@
   text-align: center;
 
   &::before {
-    content: '';
+    content: "";
```
Comment (ephys) [inline src/components/homepage-users.module.scss:49]: we should avoid making unnecessary code style changes outside of code style PRs to limit conflicts

Comment (WikiRik) [reply src/components/homepage-users.module.scss:49]: Does eslint run on scss as well? If so; can we make 1 big PR to do all style changes according to what we use for the main repo as well? If not; can we enable that and then do the same?

Comment (ephys) [reply src/components/homepage-users.module.scss:49]: It doesn't but we could set up stylelint. I don't know if it's well integrated in IDEs though. Maybe using prettier for scss would work better

Comment (WikiRik) [reply src/components/homepage-users.module.scss:49]: That might be better, @sdepold do you want to look into this?

Comment (sdepold) [reply src/components/homepage-users.module.scss:49]: Yep

**Pull request merged**